docs/users.rst

```diff
@@ -398,11 +398,18 @@ Dockerfile, the tags used within the repository are:
 
 - a unique tag including the timestamp (this tag is the only tag
   applied for scratch builds)
-- ``latest``
+- ``{version}-{release}`` (the ``version`` and ``release`` labels
+  together)
+
+If the ``tags`` value is set in container.yaml, those tags are applied
+to the image as floating tags.
+
+If ``tags`` is not used in container.yaml, the following tags are
+applied:
+
 - ``{version}`` (the ``version`` label)
-- ``{version}-{release}`` (the ``version`` and ``release`` labels together)
-- any additional tags configured in the git repository (named in the
-  ``additional-tags`` file)
+- ``latest``
+- any additional tags named in the ``additional-tags`` file
```

Comment:
Tim, I was wondering about this `additional-tags` file. Is this a legacy thing that users should not use anymore?

Reply:
It's certainly less flexible. We haven't deprecated it yet though.

Reply:
Deprecation tracked in OSBS-7698.